src/test/java/com/wikia/webdriver/common/testnglisteners/InvokeMethodAdapter.java

```diff
@@ -33,6 +37,27 @@ public void afterInvocation(IInvokedMethod method, ITestResult result) {
 
   @Override
   public void beforeInvocation(IInvokedMethod method, ITestResult testResult) {
-    // TODO Auto-generated method stub
+    if (method.isTestMethod()) {
+      if (isTestExcludedFromEnv(method.getTestMethod().getConstructorOrMethod().getMethod())) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1066)